File: 백준/Bronze/2935. 소음/소음.java
import java.math.BigInteger;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        BigInteger num1 = new BigInteger(sc.nextLine());
        String op = sc.nextLine();
        BigInteger num2 = new BigInteger(sc.nextLine());

        if (op.equals("+")) {
            System.out.println(num1.add(num2));
        } else if (op.equals("*")) {
            System.out.println(num1.multiply(num2));
        }
    }
}

File: 백준/Bronze/2935. 소음/소음.cs
using System;
using System.Numerics;

BigInteger num1 = BigInteger.Parse(Console.ReadLine());
string op = Console.ReadLine();
BigInteger num2 = BigInteger.Parse(Console.ReadLine());

if(op == "+")
{
    Console.WriteLine(num1 + num2);
}

else if(op == "*")
{
    Console.WriteLine(num1 * num2);
}